### PR TITLE
increase date container width

### PR DESCRIPTION
**Pull request by tonypls**
## Issue
Fixes
image

### CODE REVIEW

**Comment by madsnedergaard** [comment]
Nice! Did you test with other languages? German tends to always be even longer 😅

File: web/src/features/charts/tooltips/CarbonChartTooltip.tsx [hunk 1]
```diff
@@ -35,7 +35,7 @@ export default function CarbonChartTooltip({ zoneDetail }: InnerAreaGraphTooltip
           ></div>
           <div className="text-base font-bold">{__('tooltips.carbonintensity')}</div>
         </div>
-        <div className="my-1 h-[32px] max-w-[160px] select-none rounded-full bg-brand-green/10 py-2 px-3 text-sm text-brand-green dark:bg-gray-700 dark:text-white">
+        <div className="my-1 h-[32px] max-w-[165px] select-none rounded-full bg-brand-green/10 py-2 px-3 text-sm text-brand-green dark:bg-gray-700 dark:text-white">
```
Comment (VIKTORVAV99) [inline web/src/features/charts/tooltips/CarbonChartTooltip.tsx:38]: Can't we just set the width to min-content instead and skip the max width?
Or do we want to keep the same width all the time?

**Comment by madsnedergaard** [comment]
Nice! Did you test with other languages? Some might be even longer 😅

**Comment by tonypls** [comment]
> Nice! Did you test with other languages? Some might be even longer 😅

No good deed goes unpunished 😆

**Review by madsnedergaard — APPROVED** [review]
👏

**Pull request merged**